src/map/utils/synthutils.cpp

```diff
@@ -130,7 +130,7 @@ namespace synthutils
      *                                                                           *
      ****************************************************************************/
 
-    double getSynthDifficulty(CCharEntity* PChar, uint8 skillID)
+    int16 getSynthDifficulty(CCharEntity* PChar, uint8 skillID)
```

Comment:
You're changing this return from float to int, but all other functions that call this are float.  Might want to check into usage here.

Reply:
This function is called 2 times in the whole src folder.
I did change to int16 the 2 variables that store the result of this function.